network-api/networkapi/wagtailpages/factory/buyersguide.py

```diff
@@ -414,10 +423,23 @@ def generate(seed):
             page=pni_homepage,
             article=article,
         )
-
-    # Featured product updates
+    # Buyersguide featured product updates
     for update in get_random_objects(pagemodels.Update, exact_count=3):
         BuyersGuidePageFeaturedUpdateRelationFactory(
             page=pni_homepage,
             update=update,
         )
+
+    for product in pagemodels.ProductPage.objects.all():
```

Comment:
Lines 432-438 are looping through all products created by factory, and adding 5 related articles.

Lines 440-445 are looping through all product categories, and adding 1 to 6 (number chosen at random) related articles